node-scripts/spec/pipeline3/scripts.spec.js

```diff
@@ -517,4 +414,92 @@ describe("Scripts", function() {
         "\n"
     );
   });
+
+  it("should generate 'replace environment variable' script", function() {
```

Comment:
"should generate script to set or replace environment variables"

Reply:
Yes, Done